<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>380</width>
     <height>480</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgba(16,16,16,255);
border-radius:20px;</string>
   </property>
   <widget class="QLineEdit" name="LogInUser">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>240</y>
      <width>301</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(75,75,75,200);
color:rgb(0,255,0);
border-radius:2px;
border:1px solid rgba(0,0,0,0);
border-bottom-color:rgba(75,150,75,255);
padding-left:10px;
padding-bottom:3px;
</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="placeholderText">
     <string>Enter The Username</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LogInPassword">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>300</y>
      <width>301</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(75,75,75,200);
color:rgb(0,255,0);
border-radius:2px;
border:1px solid rgba(0,0,0,0);
border-bottom-color:rgba(75,150,75,255);
padding-left:10px;
padding-bottom:3px;
</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>Enter The Password</string>
    </property>
   </widget>
   <widget class="QPushButton" name="LogInButton">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>360</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Leelawadee UI</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#LogInButton{
background-color:rgba(0,240,100,255);
color:rgb(0,0,0);
border-radius:5px;
}

QPushButton#LogInButton:pressed{
background-color:rgba(0,255,150,255);
}
</string>
    </property>
    <property name="text">
     <string>Log In</string>
    </property>
   </widget>
   <widget class="QLabel" name="LockImage">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>381</width>
      <height>181</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>100</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgba(0,255,0,255);
align:center</string>
    </property>
    <property name="text">
     <string>î°ƒ</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="LogInRegisterButton">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>400</y>
      <width>371</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#LogInRegisterButton{
color:rgb(0,255,0);
}

QPushButton#LogInRegisterButton:pressed{
color:rgba(0,200,100,200);
}</string>
    </property>
    <property name="text">
     <string>Register</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="EmpCheckBoxLog">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>430</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 255, 0);</string>
    </property>
    <property name="text">
     <string>Are you an employee?</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Close">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>20</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Leelawadee UI</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#Close{
background-color:rgba(240,0,0,255);
color:rgb(0,0,0);
border-radius:5px;
}

QPushButton#Close:pressed{
background-color:rgba(255,0,0,255);
}
</string>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
